specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -363,6 +363,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/detailedusages": {

```

Comment:
>detailedusages [](start = 102, length = 14)

This name is very confusing, as api below this (GET /usages) returns usage of totally different resources - "AML resources"

With 'GET /detailedUsages' I would expect to mtch '/usages' response, with lot of additional detail
But it returns 'VM Family detailed workspace and cluster usages'

please rename this as it doesn't have context of the resource for which usages is being reported.